src/components/nav/Navbar.js

```diff
@@ -92,16 +97,18 @@ Navbar.propTypes = {
   /**
    * Fix the navbar's position at the top or bottom of the page, options: top, bottom
    */
-  fixed: PropTypes.string,
+  fixed: PropTypes.oneOf(['top', 'bottom']),
 
   /**
-   * Stick the navbar to the top or the bottom of the viewport, options: top, bottom
+   * Position the navbar at the top of the viewport, but only after scrolling past it.
+   * A convenience prop for the sticky-top positioning class. Not supported in <= IE11
+   * and other older browsers
    *
    * With `sticky`, the navbar remains in the viewport when you scroll. By
    * contrast, with `fixed`, the navbar will remain at the top or bottom of
-   * the page.
+   * the page.  sticky='top'
    */
-  sticky: PropTypes.string,
+  sticky: PropTypes.oneOf(['top']),
```

Comment:
is `'bottom'` not also an option?

Reply:
No, it's actually not an option in either [Bootstrap 4](https://getbootstrap.com/docs/4.0/components/navbar/#placement) or [Bootstrap 5](https://getbootstrap.com/docs/5.0/components/navbar/#placement).

I tried it in an app and if you use `sticky="bottom"` the class shows up in the DOM but it has no effect

Reply:
Got it, thanks! Seems strange that it's not just a boolean then 😛 

Let's leave it like this though to minimise breaking changes.

Reply:
So you mean it like `sticky: PropTypes.string,`   (and update the documentation so it doesn't mention bottom)
Right a boolean would be better.

Reply:
Yeah leave it as a string since that's what it was before. Perhaps the maintainers of reactstrap and react-bootstrap made that choice in order to allow for `"bottom"` in future?